src/index.ts

```diff
@@ -1,7 +1,7 @@
-import {Signer, providers, ContractTransaction} from "ethers";
-import {UpgradableDocumentStoreFactory} from "./contracts/UpgradableDocumentStoreFactory";
-import {DocumentStoreCreatorFactory} from "./contracts/DocumentStoreCreatorFactory";
-import {getDocumentStoreCreatorAddress} from "./config";
+import { Signer, providers, ContractTransaction } from "ethers";
+import { UpgradableDocumentStore__factory as UpgradableDocumentStoreFactory } from "./contracts/";
```

Comment:
there is only one giant file now ?

Reply:
(can combine imports)

Reply:
> there is only one giant file now ?

Yes. But you can still import directly from the `/factories` subdirectories.

> (can combine imports)

Sure, I can combine it into a single line.

Reply:
Can you check how we use this in other project (CLI, tt) i suspect a breaking change

Reply:
what's with the __ ?

Reply:
@rjchow it's because of a change in naming convention by typecheck: https://github.com/Open-Attestation/document-store/pull/82#issuecomment-829066333